lib/src/sdk/trace/span_processors/span_processor.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021-2022 Workiva.
+// Licensed under the Apache License, Version 2.0. Please see https://github.com/Workiva/opentelemetry-dart/blob/master/LICENSE for more information
+
+import '../../../../api.dart' as api;
+import '../../../../sdk.dart' as sdk;
+
+abstract class SpanProcessor {
```

Comment:
Not actionable, just thinking out loud: wondering how helpful it'd be to link to specification 🤔 

Again, not actionable here. But something like the following (or similar if there's a more appropriate syntax for dart doc comments):

```suggestion
/// Processes [api.Span]s on start and end.
///
/// * [Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#span-processor)
abstract class SpanProcessor {
```

Reply:
🤔 Might be a bit of a pain to keep up-to-date, but it'd also be handy to not have to go searching for the spec.

Reply:
Yeah, the spec docs have moved around a bit recently. Maybe the live pages have settled a bit at least 🤞 : https://opentelemetry.io/docs/specs/otel/trace/sdk/#span-processor